CHANGELOG.md

```diff
@@ -2,9 +2,24 @@
 
 ## Unreleased
 
+## v0.91.2
+
 ### 🛑 Breaking changes 🛑
+- (Splunk) - `ecs-metadata` sync the `known_status` property on the `container_id` dimension instead of lower cardinality `container_name`. This can be prevented by configuring `dimensionToUpdate` to `container_name` ([#4091](https://github.com/signalfx/splunk-otel-collector/pull/4091))
+- (Splunk) Removes `collectd/disk` monitor ([#3998](https://github.com/signalfx/splunk-otel-collector/pull/3998))
```

Comment:
need to offer notes on the migrations for all removals.

Reply:
```suggestion
- (Splunk) Removes `collectd/disk` monitor ([#3998](https://github.com/signalfx/splunk-otel-collector/pull/3998))
   This monitor has been deprecated in favor of the `disk-io` monitor.
   Note that the `disk-io` monitor has a different dimension (`disk` 
   instead of `plugin_instance`) to specify the disk.
```